.circleci/config.yml

```diff
@@ -54,8 +54,16 @@ jobs:
       - db-setup
       - ruby/rubocop-check
       - run:
-          # Ensure sample_data runs without issues
-          name: Seed with sample data
+          # Ensure db:migrate runs without issues
+          name: Migrate database
+          command: bin/rails db:migrate
+      - run:
+          # Ensure db:seed runs without issues
+          name: Load seeds
+          command: bin/rails db:seed
```

Comment:
We're not seeding before RSpec anymore, but it is still important to test that seeds work. So I added this extra step to the `static_analysis` job (which is kind of a misnomer).